components/sql/src/test/scala/pl/touk/nussknacker/sql/service/DatabaseLookupLiteRuntimeTest.scala

```diff
@@ -58,12 +58,12 @@ class DatabaseLookupLiteRuntimeTest
         "sql-lookup-enricher",
         "output",
         "sql-lookup-enricher",
-        "Table"      -> "'PERSONS'",
-        "Key column" -> "'ID'",
-        "Key value"  -> "#input",
-        "Cache TTL"  -> ""
+        "Table"      -> "'PERSONS'".spel,
```

Comment:
I wonder, why just not pass "PERSON" and let's do this `'` magic in .spel? :)

Reply:
I think it's not possible because we pass every expression to spel as String and we have to have a way to distinguish between a string and function or variable

Reply:
See `LiteralSpEL`

Reply:
What about that expression: `"#input != null".spel`? This expression cannot be quoted and LiteralSpEL will quote this. IMHO what we can do is to add another method `asStringSpel` which will add quote to passed string.